hw/ip/alert_handler/data/alert_handler.hjson.tpl

```diff
@@ -17,9 +17,12 @@ chars = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H']
 %>
 {
   name: "ALERT_HANDLER",
-  clock_primary: "clk_fixed",
+  clock_primary: "clk_i",
   bus_device: "tlul",
   regwidth: "32",
+##############################################################################
+  available_input_list: []
```

Comment:
are these required if empty?

Reply:
I think not necessarily. I can remove it.